src/components/artifacts/ArtifactsView.tsx

```diff
@@ -168,7 +191,7 @@ function ArtifactsView(props: Props) {
             <ListItemIcon>
               <InsertDriveFile />
             </ListItemIcon>
-            <ListItemText primary={info.path} />
+            <ListItemText primary={info.path} secondary={info.human_size} />
```

Comment:
Probably can do `bytesToHumanReadable(info.size)` right away without a need for `human_size`. What do you think?

Reply:
That would make more sense yeah. Web dev isn't really my area of expertise, so pardon if there are a few things that are usually done in better ways.

Reply:
Not my cup of tea as well. 😅  No worries and thank you for the contribution!

Reply:
My pleasure!